app/services/integrations/aggregator/send_private_app_token_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Integrations
+  module Aggregator
+    class SendPrivateAppTokenService < BaseService
```

Comment:
Can we also add spec for this service, pls?

Reply:
Sure forgot that one 👍